skabase/SKATestDevice/SKATestDevice/SKATestDevice.py

```diff
@@ -179,11 +197,73 @@ def RunGroupCommand(self, argin):
         return argout
         # PROTECTED REGION END #    //  SKATestDevice.RunGroupCommand
 
+    def devLogMsg(self, dev_log_msg, dev_log_level):
+        #Element Level Logging
```

Comment:
pep8

Reply:
done

Reply:
Not done.  Block comments must have a one space after the `#`.

The `flake8` tool is highly recommended.  Either run it from command line, or better yet, add it to your editor - that makes it really easy to spot these problems long before a PR.

Reply:
Yes I will add it in the editor itself. Quite helpful.